src/main/java/io/vertx/codegen/ClassModel.java

```diff
@@ -522,6 +530,11 @@ private void determineApiTypes() {
         flatMap(Helper.instanceOf(ClassTypeInfo.class)).
         filter(t -> t.getKind() == ClassKind.DATA_OBJECT).
         forEach(referencedDataObjectTypes::add);
+    collectedTypes.stream().
```

Comment:
should build rather an immutable collection rather than using foreach to add to a mutable container

Reply:
I just followed what the other streaming operations were doing, but I'm not sure if I understand, would you prefer:

```java
referencedEnumTypes = collectedTypes.stream().
      map(ClassTypeInfo::getRaw).
      flatMap(Helper.instanceOf(EnumTypeInfo.class)).
      filter(t -> t.getKind() == ClassKind.ENUM).
      collect(Collectors.toSet());
```

instead? (no `forEach` but a `collect`)?

Reply:
yes I would prefer that